website/docs/cli/index.mdx

```diff
@@ -7,7 +7,7 @@ description: >-
 
 # Terraform CLI Documentation
 
-> **Hands-on:** Try the [Terraform: Get Started](https://learn.hashicorp.com/collections/terraform/aws-get-started?utm_source=WEBSITE&utm_medium=WEB_IO&utm_offer=ARTICLE_PAGE&utm_content=DOCS) collection on HashiCorp Learn.
+> **Hands-on:** Try the [Terraform: Get Started](https://learn.hashicorp.com/collections/terraform/aws-get-started?utm_source=WEBSITE&utm_medium=WEB_IO&utm_offer=ARTICLE_PAGE&utm_content=DOCS) tutorials.
 
 This is the documentation for Terraform CLI. It is relevant to anyone working
 with Terraform's CLI-based workflows; this includes people who use Terraform CLI
```

Comment:
you know what, I wanna redo this entire intro page. I don't just wanna fix this one off without fixing the rest of it because it's the entry. If I had the time and bandwidth in this PR I'd probably rewrite all of this :) i'm going to leave this but i want to come back to this!